pkg/rbac/testdata/out

```diff
@@ -0,0 +1,83 @@
+Error: /usr/bin/gopackagesdriver: exit status 2: runtime: running GoogleExitFunction
```

Comment:
Do we need this?

Reply:
Oops. Checked this in by accident. 
Removed it.